lib/matplotlib/tests/test_colorbar.py

```diff
@@ -298,6 +298,26 @@ def test_colorbar_ticks():
     assert len(cbar.ax.xaxis.get_ticklocs()) == len(clevs)
 
 
+@cleanup
+def test_colorbar_get_ticks():
+    # test feature for #5792
+    plt.figure()
+    data = np.arange(1200).reshape(30, 40)
+
+    plt.subplot()
+    plt.contourf(data)
+
+    userTicks = plt.colorbar(use_gridspec=True, ticks=[0, 600, 1200])
```

Comment:
Is the `use_gridspec` kwarg significant or needed in any way?  If not, I would omit it.